src/org/opendatakit/briefcase/model/form/FormMetadata.java

```diff
@@ -22,22 +25,28 @@
   private final boolean hasBeenPulled;
   private final Cursor cursor;
   private final Optional<SubmissionExportMetadata> lastExportedSubmission;
+  /**
+   * The form versions of the submissions associated with this form version. Briefcase only keeps track of the currently-published form version. However,
+   * there could be form submissions that point to previous form versions.
+   */
+  private final Optional<Set<String>> submissionVersions;
```

Comment:
`Set<String>` Should be enough. Wrapping that in an `Optional` makes no sense because an empty Set would do the trick.